reference/shipping.v3.yml

```diff
@@ -410,6 +410,16 @@ components:
           example: true
         hs_codes:
           $ref: '#/components/schemas/harmonizedSystemCodes'
+        createdAt:
+          type: string
+          description: Date and time of the customs information's creation. Read-Only.
+          format: date-time
+          example: 2022-09-21T14:15:00Z
+        updatedAt:
+          type: string
+          description: Date and time when the customs information was last updated. Read-Only.
+          format: date-time
+          example: 2022-09-21T14:15:00Z
```

Comment:
Interesting - I wonder if the Tax Properties docs are incorrect here. Any thoughts @bc-andreadao ?

`2022-07-21T19:33:57+00:00` (Tax Properties)
`2018-05-07T20:14:17Z` (Tax Classes)
`2022-09-21T14:15:00Z` (Here)

Reply:
@theromulans I'm actually getting `2022-09-23T21:50:38+00:00` format for all 3 APIs, so I changed it here and created a new ticket for tax classes. Thanks for catching that!

Reply:
Just wanted to point out that `Z` and +`00:00` are essentially the same thing, but good to reflect what we're producing from an API standpoint.

Reply:
Sounds good. If you need a PR to follow, I made changes [here](https://github.com/bigcommerce/api-specs/pull/1015)